service/migration_manager.cc

```diff
@@ -726,7 +727,8 @@ future<std::vector<mutation>> prepare_column_family_update_announcement(storage_
 #endif
     try {
         auto& db = sp.local_db();
-        auto&& old_schema = db.find_column_family(cfm->ks_name(), cfm->cf_name()).schema(); // FIXME: Should we lookup by id?
+        const auto& cf = db.find_column_family(cfm->id());
```

Comment:
I see you fixed the FIXME while at it :-) I don't remember why we had this FIXME instead of just doing it...

Reply:
me neither.  I can open-code find_column_family here and lookup the table_id by `(ks_name, cf_name)`.
I don't think it can be different than `cfm->id()`.  Id that case, it would mean a new version of the table, not of its schema (i.e. after drop and recreate with the same name).
Maybe historically we may have thought the id would denote the schema version?

Reply:
left as is in next version